include/tiramisu/core.h

```diff
@@ -739,7 +778,7 @@ class function
      * Names starting with _ are reserved names.
      */
     function(std::string name);
-
+	
```

Comment:
Can you please remove the tab in this new line? Look how it appears in green here.